src/rtp/id.rs

```diff
@@ -161,6 +161,19 @@ num_id!(SessionId, u64);
 /// An RTP packet is identified by: SSRC + SeqNo. However in the RTP header the sequence number
 /// is a `u16`, meaning the value quite quickly "rolls over". To uniquely identify a packet,
 /// str0m keeps track of the roll overs and converts the `u16` to `u64` in this `SeqNo`.
+///
+/// To get the RTP u16 value from a `SeqNo`, deref and cast it.
+///
+/// ```
+/// # use str0m::rtp::SeqNo;
+/// let seq_no: SeqNo = 65_537.into();
+///
+/// // Discard upper 48 bits to get RTP u16.
+/// let rtp_seq_no = *seq_no as u16;
```

Comment:
For symmetry should we have a method that does this too?

Reply:
Give me a name. I tried to come up with one :)

Reply:
What, you don't like `seq_no.seq_no()`? 😂 

`as_u16`, `truncated_u16`, `truncated`, `without_roc`, `omitting_roc`